src/tools/kata-ctl/src/main.rs

```diff
@@ -22,11 +23,32 @@ use ops::check_ops::{
 use ops::env_ops::handle_env;
 use ops::exec_ops::handle_exec;
 use ops::volume_ops::handle_direct_volume;
+use slog::{error, o};
+
+macro_rules! sl {
+    () => {
+        slog_scope::logger().new(o!("subsystem" => "main"))
```

Comment:
why is this subsystem name main

Reply:
This I could use your help with - we named the "subsystems" whatever file they were in. I'm going to change this to `kata-ctl_main`, but without understanding how this logging is used elsewhere I'm not sure what values would be most useful.